sos/plugins/openvswitch.py

```diff
@@ -42,6 +42,18 @@ def setup(self):
             "/run/openvswitch/ovs-vswitchd.pid"
         ])
 
+        # Collect ovs-vswitchd's pmd thread details in procfs.
+        proc_files = ["environ", "cgroup", "maps", "numa_maps", "limits",
+                      "status"]
+        try:
+            pid = open("/var/run/openvswitch/ovs-vswitchd.pid").read()\
+                    .splitlines()[0]
```

Comment:
Like with #1322, this could potentially leave an open file object. Let's wrap this in a `with` clause, E.G.

```python
with open("/var/run/openvswitch/ovs-vswitchd.pid", 'r'), as pidfile:
    pid = pidfile.readline()
    self.add_copy_spec([...])
```